config/routes.rb

```diff
@@ -14,6 +14,9 @@
   delete "/api/v1/merchants/:id", to: "api/v1/merchants#destroy"
   post "/api/v1/merchants", to: "api/v1/merchants#create"
 
+  #merchant invoices
+  get "/api/v1/merchants/:merchant_id/invoices", to: "api/v1/merchants_invoices#index"
```

Comment:
Nice job with your get "/api/v1/merchants/:merchant_id/invoices". 

 It would also likely work for you to do, to: "api/v1/invoices#index" but if the merchants_invoices is getting your expected results lets go with it!